spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js

```diff
@@ -54,12 +57,46 @@ export default class DialPad extends React.Component {
             formattedPhone: ''
         };
 
-        this._asYouType = new AsYouType('US');
+        this._createAsYouType(props.countryCode);
```

Comment:
nit: maybe add a default prop of US? It's more reacty to declare the default, I think.

Reply:
ok